QuadradiusR/scripts/embedded_server.gd

```diff
@@ -0,0 +1,87 @@
+extends Node
+
+var SERVER_PORT_FILE = '.server_port'
```

Comment:
I don't really like the case used here, since we mostly use it for consts, but I see the logic behind it. We can try to think about it or it can stay like this

Reply:
I use `const` now